tensorflow/python/ops/variable_scope.py

```diff
@@ -683,6 +683,8 @@ def _get_single_variable(self,
         init_val = initializer
         variable_dtype = None
       else:
+        if type(initializer)==type:
```

Comment:
It is for preventing the exceptional illegal call of "initializer.\_\_init\_\_(shape, dtype, partition_info)" appears when the param "initializer" is an init_ops class instead of its instance.
For example, when I was using "tf/models/slim/train_image_classifier.py" to run VGG-16, the script actually passed the **type object** of class "tensorflow.python.ops.init_ops.Zeros" instead of an  **instance** of class Zeros. Hence, type(initializer) would resulted in <type 'type'>. 
This might be an out-dated usage or a minor exception. If so, this commit would not be necessary.